packages/suite/src/components/wallet/Pagination/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from 'react';
+import { colors, variables } from '@trezor/components';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin-top: 10px;
+    flex-wrap: wrap;
+`;
+
+const PageItem = styled.div<{ isActive?: boolean }>`
+    cursor: pointer;
+    background: ${props => (props.isActive ? colors.GREEN_PRIMARY : 'transparent')};
+    color: ${props => (props.isActive ? colors.WHITE : colors.GREEN_PRIMARY)};
+    padding: 4px 8px;
+    border-radius: 2px;
+`;
+
+const Actions = styled.div<{ isActive: boolean }>`
+    display: flex;
+    visibility: ${props => (props.isActive ? 'auto' : 'hidden')};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+interface Props {
+    currentPage: number;
+    totalPages?: number;
+    isOnLastPage?: boolean;
+    onPageSelected: (page: number) => void;
+}
+
+const Pagination = ({ currentPage, totalPages, onPageSelected, isOnLastPage }: Props) => {
+    // if totalPages is undefined show only start/prev/next buttons
+    const showNext = totalPages ? currentPage < totalPages : !isOnLastPage;
+    const showPrevious = currentPage > 1;
+    // array of int used for creating all page buttons
+    const calculatedPages = useMemo(() => [...Array(totalPages)].map((_p, i) => i + 1), [
+        totalPages,
+    ]);
+
+    return (
+        <Wrapper>
+            <Actions isActive={showPrevious}>
+                <PageItem onClick={() => onPageSelected(1)}>«</PageItem>
+                <PageItem onClick={() => onPageSelected(currentPage - 1)}>‹</PageItem>
+            </Actions>
```

Comment:
shoudn't we use html entities istead of unicode chars?

`&laquo;` or `&#171;`

Reply:
https://shripadk.github.io/react/docs/jsx-gotchas.html
i've found this, looks like it's ok this way, ignore this :)